README.md

```diff
@@ -2,7 +2,7 @@
 >
 > As this is a proof of concept,  it won't be supported by the k6 team.
 > It may also break in the future as xk6 evolves. USE AT YOUR OWN RISK!
-> Any issues with the tool should be raised [here](https://github.com/lxkuz/xk6-amqp/issues).
+> Any issues with the tool should be raised [here](https://github.com/k6io/xk6-amqp/issues).
```

Comment:
```suggestion
> Any issues with the tool should be raised [here](https://github.com/grafana/xk6-amqp/issues).
```

Reply:
@simskij does it mean I should use `github.com/grafana` instead of `github.com/k6io` **everywhere**?

Reply:
Yes, correct